js/views/modals/Settings/Page.js

```diff
@@ -30,9 +30,54 @@ export default class extends baseVw {
       'click .js-avatarRight': 'avatarRightClick',
       'click .js-headerLeft': 'headerLeftClick',
       'click .js-headerRight': 'headerRightClick',
+      'change .js-colorPicker': 'handleColorChosen',
+      'change .js-colorCode': 'handleColorCodeEntered',
     };
   }
 
+  /** Handles when a hex color code is entered by updating color picker. */
+  handleColorCodeEntered(event) {
+    const colorPickerId = $(event.target).data('color-picker-id');
+    const $colorPicker = this.getCachedElement(colorPickerId);
+    const newHexColorCode = event.target.value;
+
+    // If the text passes a basic RegExp for a valid 6 digit hex value,
+    // update the color picker's color.
+    if (/^#([0-9a-f]{6})$/i.test(newHexColorCode)) {
+      $colorPicker.val(newHexColorCode);
+    }
+  }
+
+  /** Handles when a color is chosen from the color picker by updating hex color code text. */
+  handleColorChosen(event) {
+    const hexInputId = $(event.target).data('hex-input-id');
+    const $hexInput = this.getCachedElement(hexInputId);
+    const newColor = event.target.value;
+
+    $hexInput.val(newColor);
+  }
+
+  /** Returns a given jquery Element or a locally cached version in this view for optimization. */
+  getCachedElement(elementId) {
+    let element;
+
+    // Ensure we have our cached elements map.
+    if (!this._cachedElementMap) {
+      this._cachedElementMap = new Map();
+    }
+
+    // If the cache has the element, we shall use it.
+    if (this._cachedElementMap.has(elementId)) {
+      element =  this._cachedElementMap.get(elementId);
+    } else {
+      // The cache does not not have the element, therefore query with jQuery and cache it.
+      element = $(elementId);
```

Comment:
Could we scope the jQuery query to this views container? The Backbone.View contains a convenience method to do just that `this.$('<selector>')` which is short-hand for `$(view.el).find(<selector>)`.

It's generally the best practice to find elements of the view using that approach. Some of the benefits are:
- You're querying only a sub-set of the DOM instead of the entire DOM.
- If there's another element in the DOM outside of this view matched by the same selector, you won't unintentionally target it.